recipe/recipe.yaml

```diff
@@ -0,0 +1,83 @@
+context:
+  name: rustworkx
+  python_name: rustworkx
+  version: "0.16.0"
+  python_min: "3.9"
+
+package:
+  name: rustworkx
+  version: ${{ version }}
+
+source:
+  url: https://github.com/Qiskit/${{ name }}/archive/refs/tags/${{ version }}.tar.gz
+  sha256: ef3c6fb3bd7e6a18919023330f3961702346d8afbe95aa5f446f9b222ee72391
+
+build:
+  number: 2
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - ${{ PYTHON }} -m pip install . -vv
+  skip: not (match(python, python_min ~ ".*") and is_abi3)
+  python:
+    version_independent: true
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+      - python
+      - cross-python_${{ target_platform }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - python-abi3
+    - pip
+    - setuptools
+    - setuptools-rust
+    - wheel
```

Comment:
Comparing to the upstream pyproject.toml, I see we could put lower bounds on setuptools and setuptools-rust and also might be able to drop wheel.

Reply:
Once we release 0.17 I can tell you for sure we’ll need higher bounds on setup-tools as we moved to pyproject.toml.

for 0.16 we still use the legacy setup.py style of config. So I think you don’t need to update it now

Reply:
Removing wheel though. My understanding is that it was cargo culted for a while, it should be safe to ax

Reply:
In practice, the CI installs into a clean environment each time so putting a lower bound on the build environment packages doesn't do much but I added them just so we match upstream as much as possible.